### PR TITLE
Project: Bare Metal Forms and Helpers 

### PR DESCRIPTION
**Pull request by edieatha**
✔️ Set up the back end with rails
✔️ Set up the HTML Form
✔️ Rails Forms with #form_tag
✔️ Rails Forms with #form_for

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username],email: params[:email], password: params[:password])
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: There should be no commented out code.
lease remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag: 

Hi Team! :wave: 

Great job! Everything looks fine. :slightly_smiling_face: 

Simply remove commented out code (other than the forms lol) and you should be fine to move on!

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+  def update
+    @user = User.find(params[:id])
+    if @user.save
+        @user.update(user_params)
+        redirect_to new_user_path
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: This should be `edit_user_path`

**Comment by edieatha** [comment]
✅ Removed commented code from users_controller
✅ Fixed edi_user_path instead for new_user_path in #edit

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Nice work on this project, but the following changes need to be done before moving on ⬇️** 

- The linter check is not activated on the repo, please activate it and make it pass.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by edieatha** [comment]
✔️ Added stickler fixed linter errors

**Review by mosesogwo — APPROVED** [review]
**:clap:Hello!, I see your linter check now:+1:. You are now good to go :rocket:**

### Status: Approved :heavy_check_mark:

**Pull request merged**